cmd/crane/cmd/root.go

```diff
@@ -80,3 +87,31 @@ func (ht *headerTransport) RoundTrip(in *http.Request) (*http.Response, error) {
 	}
 	return ht.inner.RoundTrip(in)
 }
+
+type platformValue struct {
+	platform *v1.Platform
+}
+
+func (pv *platformValue) Set(platform string) error {
```

Comment:
We'll probably also want to allow a third value for the variant. We do this in [ko](https://github.com/google/ko/blob/9b910355628d9f9bb7a386fd42608ff4dc34c4a4/pkg/commands/config.go#L75-L87) and I'd like to be consistent.

[containerd](https://github.com/containerd/containerd/blob/ba8377590d326a5f73e4abe2d282c46c385329ae/platforms/platforms.go#L69) also allows this, and in the fullness of time I'd like to be compatible with what they allow, but for now let's just do the os/arch/variant form.